.gitpod.yml

```diff
@@ -0,0 +1,53 @@
+image:
+  file: .gitpod.Dockerfile
+
+# Adding vscode C# extension
+vscode:
+  extensions:
+    - muhammad-sammy.csharp
+
+# Ports to expose
+ports:
+  - name: Consul Dev Server UI
+    port: 8500
+    onOpen: open-preview
+
+# Start-up tasks
+tasks:
+  - name: Consul.NET Client
+    before: |
+      dotnet --version
+    init: |
+      dotnet build
+      cd Consul.Test/ && pwd
+      dotnet build
+      cd ..
+    command: |
+      cd Consul.Test/ && pwd
+      dotnet run --framework "net6.0"
+      cd ..
```

Comment:
I don't understand why this task is necessary, without it I was unable to run tests from the VsCode ("Failed to run test because "System.ArgumentNullException: Value cannot be null. (Parameter 'workingDirectory')\n at...").

Reply:
Indeed, these commands aren't necessary, I put them just as an example.
Can you please give me more details about this error and how I can reproduce it.

Reply:
![image](https://user-images.githubusercontent.com/2308005/209106050-f5641987-7994-48bd-b4e2-d9225231f5fe.png)

Reply:
![Screenshot 2022-12-22 at 10 49 46](https://user-images.githubusercontent.com/18899702/209107123-2e34a282-241f-4853-9d98-b1a3a260762c.png)

Reply:
Can you please expand the error so we can see the details?

Reply:
> I don't know why I didn't get an error, or maybe you disabled the task above ?

Yes, I get this error only when I remove "Consul.NET Client" task. The error says:
```
Failed to run test because "System.ArgumentNullException: Value cannot be null. (Parameter 'workingDirectory')
   at OmniSharp.DotNetTest.TestManager..ctor(Project project, String workingDirectory, IDotNetCliService dotNetCli, SemanticVersion dotNetCliVersion, IEventEmitter eventEmitter, ILogger logger) in /Users/runner/work/1/s/src/OmniSharp.DotNetTest/TestManager.cs:line 45
   at OmniSharp.DotNetTest.TestManager.Create(Project project, IDotNetCliService dotNetCli, IEventEmitter eventEmitter, ILoggerFactory loggerFactory) in /Users/runner/work/1/s/src/OmniSharp.DotNetTest/TestManager.cs:line 70
   at OmniSharp.DotNetTest.Services.RunTestsInContextService.Handle(RunTestsInContextRequest request) in /Users/runner/work/1/s/src/OmniSharp.DotNetTest/Services/RunTestsInContextService.cs:line 29
  at OmniSharp.Endpoint.EndpointHandler`2.GetFirstNotEmptyResponseFromHandlers(ExportHandler`2[] handlers, TRequest request) in /Users/runner/work/1/s/src/OmniSharp.Host/Endpoint/EndpointHandler.cs:line 201
   at OmniSharp.Endpoint.EndpointHandler`2.HandleRequestForLanguage(String language, TRequest request, RequestPacket packet) in /Users/runner/work/1/s/src/OmniSharp.Host/Endpoint/EndpointHandler.cs:line 234
   at OmniSharp.Endpoint.EndpointHandler`2.Process(RequestPacket packet, LanguageModel model, JToken requestObject) in /Users/runner/work/1/s/src/OmniSharp.Host/Endpoint/EndpointHandler.cs:line 143
   at OmniSharp.Stdio.Host.HandleRequest(String json, ILogger logger) in /Users/runner/work/1/s/src/OmniSharp.Stdio/Host.cs:line 258".
```

Reply:
Seems to be related to VSCode Extension https://github.com/OmniSharp/omnisharp-vscode/issues/4783.
I think we can just keep the task to avoid this issue.
What do you think ?

Reply:
> Seems to be related to VSCode Extension [OmniSharp/omnisharp-vscode#4783](https://github.com/OmniSharp/omnisharp-vscode/issues/4783). I think we can just keep the task to avoid this issue. What do you think ?

I agree, I think we can just add a comment with explanation that at first glance this taks seems to be not necessary but it  actually is necessary to workaround problems with OmiSharp.